tests/bson/bson-dbpointer-tostring-001.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+MongoDB\BSON\DBPointer::__toString()
```

Comment:
This implementation appears to be missing. Was it inadvertently removed? I remember seeing a `[%s/%s]` pattern in an earlier PR diff.

Also, should we have a `__toString()` method for Undefined that returns an empty string? That would make it consistent with `(string) null`.